src/exampleTypes/Bool.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react'
-import F from 'futil'
+import _ from 'lodash/fp'
 import { contexturify } from '../utils/hoc'
 
-let Bool = ({ tree, node, theme: { RadioList } }) => (
-  <div className="contexture-bool">
-    <RadioList
-      value={node.value ? 'yes' : 'no'}
-      onChange={value => {
-        tree.mutate(node.path, { value: value === 'yes' })
-      }}
-      options={F.autoLabelOptions(['yes', 'no'])}
-    />
-  </div>
-)
+let getValue = (value, either) =>
+  _.isBoolean(value) ? value : either ? null : false
```

Comment:
This might be simpler with a single check instead of a nested ternary:

```js
let getValue = (value, either) => (_.isNil(value) && either)  ? null : !!value
```

The main implementation difference is that this new version will continue to treat truthy values as `true`, whereas the one in the PR here would treat something like `5` as `false`

Reply:
Also, it might make sense to curry and flip this, so `either` comes first, but that's up to you since this is an internal util